packages/components/popover/src/Popover.tsx

```diff
@@ -0,0 +1,240 @@
+import React, { useMemo, useState, useEffect, useCallback } from 'react';
+import { CommonProps, useId, mergeRefs } from '@contentful/f36-core';
+import { Placement, Modifier } from '@popperjs/core';
+import { PopoverContextProvider, PopoverContextType } from './PopoverContext';
+import { usePopper } from 'react-popper';
+
+export interface PopoverProps extends CommonProps {
+  children: React.ReactNode;
+
+  /**
+   * Boolean to determine if the Popover should be the same width as
+   * the trigger element
+   *
+   * @default false
+   */
+  isFullWidth?: boolean;
+
+  /**
+   * Boolean to control whether or not the Popover is open
+   *
+   * @default false
+   */
+  isOpen?: boolean;
+
+  /**
+   * Callback fired when the popover closes
+   */
+  onClose?: Function;
+
+  /**
+   * Determines the preferred position of the Popover. This position is not
+   * guaranteed, as the Popover might be moved to fit the viewport
+   *
+   * @default bottom
+   */
+  placement?: Placement;
+
+  /**
+   * Boolean to disable automatic positioning of the element to fit the
+   * available viewport. Instead this forces the element to follow the given
+   * value of the `placement` prop.
+   *
+   * @default true
+   */
```

Comment:
```suggestion
  /**
   * Popper.js will try to prevent the Popover from overflowing and overlapping (the available viewport???).
   * If this prop is set to false, it disables that behavior and Instead it forces the element to follow the given
   * value of the `placement` prop, even if it's overlapped.
   *
   * @default true
   */
```

Something like this?????? don't trust me on writing text honestly

Reply:
Is it better?
https://github.com/contentful/forma-36/pull/1204/commits/af1013eddb2e786463a43dad8ccb693b149be65f#diff-312f071358cb2435722193ae98706bfa3b73a5ea7b43eddf8f5e2ed66957146dR39-R46